src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -29888,7 +29888,7 @@ public static void TakeOutParam(out int x, long y)
     }
 }
 ";
-            var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.ReleaseExe.WithScriptClassName("Script"), parseOptions: TestOptions.Script);
+            var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.ReleaseExe.WithScriptClassName("Script"), parseOptions: TestOptions.Script, skipUsesIsNullable: true);
```

Comment:
>skipUsesIsNullable: true [](start = 167, length = 24)

What does this do and why do we need this here?

Reply:
This skips visiting symbols in the compilation in `CreateCompilation`. It's actually not needed in this test, just the subsequent tests. In those cases, if we visit symbols eagerly we'll report `ERR_RecursivelyTypedVariable` on a different local (field). (In `_03`, we report the error on `a` rather than `b` for instance.)

---
In reply to: [171141133](https://github.com/dotnet/roslyn/pull/25087#discussion_r171141133) [](ancestors = 171141133)